pwndbg/disasm/color.py

```diff
@@ -11,8 +11,11 @@
 capstone.CS_GRP_JUMP
 ))
 
-def instruction(ins):
-    asm = u'%-06s %s' % (ins.mnemonic, ins.op_str)
+def instruction(ins, highlighted=False):
+    def maybe_highlight(s):
+        return s if not highlighted else pwndbg.color.bold(pwndbg.color.green(s))
```

Comment:
This would be much simpler as:

```py
def highlight(line):
    if pwndbg.config.highlight_pc:
        return pwndbg.color.bold(pwndbg.color.green(line))
    return line
```

Also please pull the function to the outer scope.